theme/index.html

```diff
@@ -0,0 +1,120 @@
+<html>
+    <head>
+        <title>Meshnet/Documentation</title>
+        <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+        <link href="https://fonts.googleapis.com/css?family=Inconsolata" rel="stylesheet">
+        <style>
```

Comment:
Is there a reason why the style isn't in a separate file?

Reply:
For simplicity; if the style is needed for one page only, why would we waste requests to get resources that can easily be inside the html?